src/lib/y2network/proposal_settings.rb

```diff
@@ -107,6 +136,13 @@ def use_network_manager?
       network_manager_default?
     end
 
+    # Convenience method to determine if the NM package is installed or not
+    #
+    # @return [Boolean] true if NetworkManager is installed; false otherwise
+    def network_manager_installed?
+      Yast::Package.Installed("NetworkManager")
```

Comment:
I've still stronger feeling that this code duplicates anything we already have. 

For example this seems to be duplicate of ```NetworkService::backend_available?```

Reply:
Well, at the end, both are convenience method for checking if the package is installed. NetworkService make sense to me in a running system, but during installation, there is no separation between .. the proposal and the instsys network configuration That is something I do not like at all, if I just change the service at that point, the configuration module is also affected.

So tried to not depend on NetworkService and now even importing Lan can be removed.